File_Search/readme.md

```diff
@@ -0,0 +1,31 @@
+File Path Finder and Unique Extension Lister:
```

Comment:
Can you consider improving the all the heading?

Reply:
Sure, I will improve it